core/server/lib/members/static/gateway/bundle.js

```diff
@@ -23,9 +23,76 @@
         };
     }
 
+    function isTokenExpired(token) {
+        try {
+            const [header, claims, signature] = token.split('.');
+
+            const parsedClaims = JSON.parse(atob(claims.replace('+', '-').replace('/', '_')));
```

Comment:
The replace here converts from base64url to base64 encoding, and atob decodes from base64